modules/dnn/src/dnn.cpp

```diff
@@ -1263,6 +1278,12 @@ struct Net::Impl : public detail::NetImplBase
             {
                 return wrapMat(preferableBackend, preferableTarget, host);
             }
+            else if (preferableBackend == DNN_BACKEND_WEBNN)
+            {
+                #ifdef HAVE_WEBNN
```

Comment:
Please keep the indent aligned to others.

Reply:
fixed